# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../transconnect/prefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN transconnect-1.3-Beta/Makefile transconnect-1.3-Beta-new/Makefile
--- transconnect-1.3-Beta/Makefile	2002-02-06 09:45:33.000000000 +0100
+++ transconnect-1.3-Beta-new/Makefile	2004-04-15 16:34:27.738979376 +0200
@@ -1,4 +1,5 @@
 # Makefile for transconnect
+PREFIX = /usr
 
 CFLAGS  = -Wall 
 LDFLAGS = -shared
@@ -28,13 +29,10 @@
 	$(CC) $(SUN_LDFLAGS) $(SUN_LDLIBS) -o tconn.so tconn.c
 
 install:
-	mkdir -p $(HOME)/.tconn
-	chmod 700 $(HOME)/.tconn
-	cp -f tconn.so $(HOME)/.tconn/
-	- cp -f $(HOME)/.tconn/tconn.conf $(HOME)/.tconn/tconn.conf.bak 
-	cp -f tconn.conf $(HOME)/.tconn/
-	cp -f README $(HOME)/.tconn/
-	cp -f INSTALL $(HOME)/.tconn/
+	mkdir -p $(PREFIX)/tconn
+	cp -f tconn.so $(PREFIX)/tconn/
+	- cp -f $(PREFIX)/tconn/tconn.conf $(PREFIX)/tconn/tconn.conf.bak 
+	cp -f tconn.conf $(PREFIX)/tconn/
 	cat tconn.cat
 
 clean:
diff -ruN transconnect-1.3-Beta/tconn.cat transconnect-1.3-Beta-new/tconn.cat
--- transconnect-1.3-Beta/tconn.cat	2001-10-15 12:53:50.000000000 +0200
+++ transconnect-1.3-Beta-new/tconn.cat	2004-04-15 16:35:20.492959552 +0200
@@ -1,13 +1,14 @@
 
 Transconnect has been installed
 Now Edit the .tconn/tconn.conf in your home directory
+You can take a sample from /usr/tconn/tconn.conf.
 
 if you want your config file to reside at a different
 place you can set the TCONN environment variable
 export TCONN=<path>/<filename>
 
 type the following for starting transconnect
-export LD_PRELOAD=$HOME/.tconn/tconn.so
+export LD_PRELOAD=/usr/tconn/tconn.so
 
 For stopping the transconnect type
 unset LD_PRELOAD
